src/components/RedoIcon.tsx

```diff
@@ -4,19 +4,43 @@ import { token } from '../../styled-system/tokens'
 import Icon from '../@types/Icon'
 
 /** A redo icon. */
-const RedoIcon = ({ fill, size = 18, style, className }: Icon) => {
+const RedoIcon = ({ fill, size = 14, style = {}, className }: Icon) => {
```

Comment:
I think changing the default font size to 14 is going to break some of the icons that are used in multiple places, such as https://github.com/cybersemics/em/blob/main/src/components/icons/HomeIcon.tsx. The size should correspond to font size, which defaults to 18. Can you change the default size to 18 and the scale factor to 1.066, or am I missing something?

Can you change the default font size to 18 and the scale factor to 1.066, or does that mess things up?

Reply:
Scaling is a part of Milestone 3, so we can keep all default values of size for now.

Factors to Keep in Mind for Milestone 3:

Size can be set by passing the parameter when rendering the icon component.
The default font size is set to 18 in constants, and it will remain as 18, while each icon component has its own default size prop.
The HomeIcon currently doesn’t have a default size.
Undo and redo are used not only in the toolbar but also in the modal window. We can pass size={14} explicitly from the modal window component for these icons.

Reply:
Thanks! We can revisit in Milestone 3.